php/Terminus/Models/Environment.php

```diff
@@ -648,9 +648,23 @@ public function sendCommandViaSsh($command)
             'ssh -T %s@%s -p %s -o "AddressFamily inet" %s',
             [$sftp['username'], $sftp['host'], $sftp['port'], escapeshellarg($command),]
         );
+
+        // Catch Terminus running in test mode
+        $test_mode = isset($_ENV["TERMINUS_TEST_MODE"])
+            && filter_var($_ENV["TERMINUS_TEST_MODE"], FILTER_VALIDATE_BOOLEAN);
+        if ($test_mode) {
```

Comment:
Even here in `Models\Environment`? I assumed that would be well, assuming too much this deep. I'll give it a shot if that is the recommended route.